apps/test/unit/templates/rubrics/RubricContainerTest.jsx

```diff
@@ -195,17 +229,10 @@ describe('RubricContainer', () => {
   });
 
   it('switches components when tabs are clicked', async () => {
-    fetchStub
-      .onCall(0)
-      .returns(Promise.resolve(new Response(JSON.stringify(successJson))));
-    fetchStub
-      .onCall(1)
-      .returns(Promise.resolve(new Response(JSON.stringify(successJsonAll))));
-    fetchStub
-      .onCall(2)
-      .returns(
-        Promise.resolve(new Response(JSON.stringify(mockAiEvaluations)))
-      );
+    stubFetchEvalStatusForUser(successJson);
+    stubFetchEvalStatusForAll(successJsonAll);
+    stubFetchAiEvaluations(mockAiEvaluations);
```

Comment:
A lot of the changes to this test module are to get rid of the arbitrary order of fetch calls. Adding the support for another API call throughout and it being rendered as a whole and not shallowly just meant that it was hurting my head to figure out what the order was supposed to be.

So I'm stubbing out the fetches filtered by their URL so I can see that they are also being passed the right arguments and then it doesn't matter what order they happen in. I just care that they _have happened_ at each interval the result matters.